coverage/.resultset.json

```diff
@@ -0,0 +1,263 @@
+          2,
+          null,
+          null,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          null,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          null,
+          null,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null
+        ]
+      },
+      "/Users/ianmcnicholas/MAKERS/Week_5/bowling-challenge-ruby/lib/game.rb": {
+        "lines": [
+          null,
+          null,
+          1,
+          null,
+          1,
+          1,
+          null,
+          1,
+          null,
+          null,
+          null,
+          null,
+          17,
+          17,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          24,
+          24,
+          null,
+          23,
+          23,
+          16,
+          3,
+          3,
+          null,
+          13,
+          null,
+          7,
+          1,
+          6,
+          3,
+          null,
+          3,
+          null,
+          22,
+          null,
+          null,
+          null,
+          null,
+          1,
+          22,
+          22,
+          220,
+          null,
+          22,
+          null,
+          3,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          24,
+          24,
+          240,
+          null,
+          24,
+          null,
+          null,
+          24,
+          null,
+          null
+        ]
+      }
+    },
+    "timestamp": 1615152610
+  }
+}
```

Comment:
you could consider putting your coverage folder in a .gitignore file so you don't get all the empties on github